integration/spark/app/src/test/java/io/openlineage/spark/agent/ArgumentParserTest.java

```diff
@@ -99,6 +101,8 @@ void testConfToHttpConfig() {
     assert (transportConfig.getAuth() instanceof ApiKeyTokenProvider);
     assertEquals("Bearer random_token", transportConfig.getAuth().getToken());
     assertEquals(5000, transportConfig.getTimeout());
+    assertEquals("test1", transportConfig.getHeaders().get("test1"));
```

Comment:
I think it's a good practice to set different values for keys and values.

Reply:
ok I'll change that